### PR TITLE
fix #3440 (ask for broadcast name)

### LINKED ISSUE #3440: Ask for the name when creating a broadcast list
Before https://github.com/deltachat/deltachat-core-rust/pull/4644, when creating a broadcast list, the UIs didn't ask for a name since the name wasn't shown to the recipients and therefore not that important.

As of https://github.com/deltachat/deltachat-core-rust/pull/4644, broadcast lists will create their own chat for the recipients, showing the broadcast lists's name, so we need to ask the user for a name when creating a broadcast list.

See https://github.com/deltachat/deltachat-android/pull/2653/files#diff-7c8940eaea1021916d97c310d04af7ebc523920d0bb7e57399e1f7655088d527 for how this is done in Android.

### PR DESCRIPTION
**Pull request by farooqkz**
This fixes #3440

### CODE REVIEW

**Comment by Simon-Laux** [comment]
Bildschirmfoto 2023-10-31 um 18 24 00

Please note that I rebased your pr to current master, so you need to delete your local branch and pull the remote branch.

**Comment by Simon-Laux** [comment]
I fixed the bug, but could still be improved to look better
Bildschirmfoto 2023-10-31 um 18 43 39

File: src/renderer/components/dialogs/CreateChat.tsx [hunk 1]
```diff
@@ -283,69 +283,68 @@ export function useGroupMembers(initialMemebers: number[]) {
...
+            onChange={onChange}
+            autoFocus
+            spellCheck={false}
+          />
+          {errorMissingChatName && (
+            <p className='input-error'>
+              {type === 'group'
+                ? tx('group_please_enter_group_name')
+                : tx('group_please_enter_group_name')}
+            </p>
```
Comment (Simon-Laux) [inline src/renderer/components/dialogs/CreateChat.tsx:344]: we probably need a new string for broadcast lists?

File: src/renderer/components/dialogs/CreateChat.tsx [hunk 2]
```diff
@@ -283,69 +283,68 @@ export function useGroupMembers(initialMemebers: number[]) {
...
+            onSetGroupImage={onSetGroupImage}
+            onUnsetGroupImage={onUnsetGroupImage}
+            groupName={chatName}
+            color={color}
+          />
+        )}
+        <div className='group-name-input-wrapper'>
+          <input
+            className='group-name-input'
+            placeholder={type === 'group' ? tx('group_name') : tx('group_name')}
```
Comment (Simon-Laux) [inline src/renderer/components/dialogs/CreateChat.tsx:333]: here also other string for broadcast list is needed

File: src/renderer/components/dialogs/CreateChat.tsx (hunk 1 above)
Comment (farooqkz) [reply src/renderer/components/dialogs/CreateChat.tsx:344]: yeah

File: src/renderer/components/dialogs/CreateChat.tsx (hunk 2 above)
Comment (farooqkz) [reply src/renderer/components/dialogs/CreateChat.tsx:333]: yeah

**Pull request merged**